Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIRuntimeCocoa.mm

```diff
@@ -122,7 +122,8 @@
 
 void WebExtensionContext::runtimeReload()
 {
-    reload();
+    RefPtr<API::Error> outError = nullptr;
+    reload(outError);
```

Comment:
`reload()` should have a default parameter value for `outError`, so no change is needed here.